### ISSUE TITLE
vmprof example not showing jitlog stats 

### ISSUE THREAD

**Issue by zmjiang** [vmprof/vmprof-python#126]
We are trying to run the cpuburn.py example on the website with pypy version 5.7.0 (64 bit version) on Ubuntu. We used exactly the same command to generate the dat file. When we try to visualize the results with vmprofshow (see screenshot below), it did not include any jit timing information as suggested on the website. 

![image](https://cloud.githubusercontent.com/assets/5704895/24430160/4ce3f8ba-13e3-11e7-9bc0-eca3dfe2ac9d.png)

Could you please let us know if we are missing anything here?

**Comment by planrich** [comment]
I agree that some JIT code frames are missing. Never the less, the functions burn, _iterate, _next_random are jit compiled. You can see that in this profile (--web)
http://vmprof.com/#/3adcaf52-cf1c-44ed-84c2-f056d1717d76

As far as I can tell, PyPy 5.7.0 and the one I'm using which is a version after PyPy 5.7.0 do not log the JIT code entries into the profile. I'm investigating.

**Comment by zmjiang** [comment]
Thanks for the prompt response! 

We will use the web option for now. Please let us know your investigation results regarding the issues of logging JIT code entries.

**Comment by LoHiaufung** [comment]
now the Jit code stat is missing, this link is 404.
http://vmprof.com/#/3adcaf52-cf1c-44ed-84c2-f056d1717d76
'''
Http: 404 Not Found. URL: /api/jit/meta/3adcaf52-cf1c-44ed-84c2-f056d1717d76/
'''

**Comment by LoHiaufung** [comment]
Hi, planrich, is the jit code stat work now?
